.devcontainer/Dockerfile

```diff
@@ -8,6 +8,6 @@ RUN install-apt make g++
 RUN install-tool python 3.11.4
```

Comment:
I'm testing this PR in a new Codespace on your `build/npm` branch.

When the Codespace is loaded, I see the VSCode IDE. I see a terminal tab called `Jest (renovate)` which shows this message:

```
Jest (renovate)

/bin/sh: 1: yarn: not found


[error] failed to retrieve test file list. TestExplorer might show incomplete test items

[info] jest process failed to start, most likely due to env or project configuration issues, please see: https://github.com/jest-community/vscode-jest/blob/master/README.md#jest-failed-to-run
```

Looks like there's a stray `yarn` reference somewhere?